src/api/spec/controllers/webui/attributes_spec.rb

```diff
@@ -16,4 +16,42 @@
       expect(flash[:error]).to eq("Project not found: Does:Not:Exist")
     end
   end
+
+  describe 'POST #create' do
+    let(:attribute_type_0) { create(:attrib_type_with_namespace, value_count: 0) }
+    let(:attribute_type_1) { create(:attrib_type_with_namespace, value_count: 1) }
+    let(:attribute_type_1_name) { "#{attribute_type_1.namespace}:#{attribute_type_1.name}" }
+
+    before do
+      login user
+    end
+
+    context 'with editable values' do
+      before do
+        post :create, params: { attrib: { project_id: user.home_project.id, attrib_type_id: attribute_type_1.id } }
+      end
+
+      it { expect(response).to redirect_to(edit_attribs_path(project: user.home_project_name, package: '', attribute: attribute_type_1_name)) }
+      it { expect(flash[:notice]).to eq('Attribute was successfully created.') }
+    end
+
+    context 'with non editable values' do
+      before do
+        post :create, params: { attrib: { project_id: user.home_project.id, attrib_type_id: attribute_type_0.id } }
+      end
+
+      it { expect(response).to redirect_to(index_attribs_path(project: user.home_project_name, package: '')) }
+      it { expect(flash[:notice]).to eq('Attribute was successfully created.') }
+    end
+
+    context 'fails at save' do
+      before do
+        allow_any_instance_of(Attrib).to receive(:save).and_return(false)
+        post :create, params: { attrib: { project_id: user.home_project.id, attrib_type_id: attribute_type_1.id } }
+      end
+
+      it { expect(response).to redirect_to(root_path) }
+      it { expect(flash[:error]).to eq('Saving attribute failed: ') }
```

Comment:
@Ana06 In this case, 'things that can actually happen' to make this `save` call return `false` would be better tested in a model or feature test, I think.

Can we agree on that?